package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "pulltorefresh",
+  "version": "0.1.0",
+  "description": "Change a number value using a slider",
+  "scripts": {
```

Comment:
If there are no tests. remove all test files and reference like karma enzyme etc.